### ISSUE TITLE
kimchi virtualization tab is missing.

### ISSUE THREAD

**Issue by FaizanSh** [kimchi-project/kimchi#1252]
I am a beginner I have read all the comments but I couldn't figured what we need to edit in template.conf file. I have the same file as this https://github.com/kimchi-project/kimchi/blob/master/template.conf 
I am getting following errors when I see wok-error.log 
_Last portion of the output_

> *** Kimchi: Running feature tests ***
> Service Libvirtd running ...: True
> operation failed: no storage pools were found on host '127.0.0.1'
> NFS Target Probe support ...: True
> Fibre Channel Host support .: True
> Kernel VFIO support ........: True
> Network Manager running ....: True
> Memory Hotplug support .....: False
> *** Kimchi: Feature tests completed ***
> 
> Unknown device type: drm
> Unknown device type: drm
> Fatal: Unable to create storage pool default. 
> Details: cannot open directory '/var/lib/libvirt/images': No such file or directory
> Failed to import plugin plugins.kimchi.Kimchi, error: Storage pool default does not exist or is not active. Please, check the configuration in /etc/kimchi/template.conf to ensure it lists only valid storage.
> *** Ginger Base: Running capabilities tests ***
> System Report Tool ...: True
> Logging AptUpdate features.
> System Update Tool ...: True
> Repo Management Tool .: deb
> *** Ginger Base: Capabilities tests completed ***
> [19/Sep/2018:03:29:41] ENGINE Listening for SIGHUP.
> [19/Sep/2018:03:29:41] ENGINE Listening for SIGTERM.
> [19/Sep/2018:03:29:41] ENGINE Listening for SIGUSR1.
> [19/Sep/2018:03:29:41] ENGINE Bus STARTING
> [19/Sep/2018:03:29:41] ENGINE Serving on http://127.0.0.1:8010
> 
> *** Kimchi: Running dependable feature tests ***
> [19/Sep/2018:03:29:41] ENGINE Started monitor thread 'Session cleanup'.
> QEMU stream support .......: False
> Libvirt Stream Protocols ..: ['http', 'https', 'ftp', 'ftps', 'tftp']
> *** Kimchi: Dependable feature tests completed ***
> 
> [19/Sep/2018:03:29:41] ENGINE Bus STARTED
> [19/Sep/2018:03:36:01] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:36:01] ENGINE Bus STOPPING
> [19/Sep/2018:03:36:01] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) shut down
> [19/Sep/2018:03:36:01] ENGINE Stopped thread 'Session cleanup'.
> [19/Sep/2018:03:36:01] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:36:02] ENGINE Bus STOPPING
> [19/Sep/2018:03:36:02] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) already shut down
> [19/Sep/2018:03:36:02] ENGINE No thread running for Session cleanup.
> Connection to libvirt broken. Recycling. ecode: 1 edom: 7
> Connection to libvirt broken. Recycling. ecode: 1 edom: 7
> [19/Sep/2018:03:36:02] ENGINE Bus STOPPED
> [19/Sep/2018:03:36:02] ENGINE Bus EXITING
> [19/Sep/2018:03:36:02] ENGINE Bus EXITED
> [19/Sep/2018:03:36:02] ENGINE Bus STOPPED
> [19/Sep/2018:03:36:02] ENGINE Bus EXITING
> [19/Sep/2018:03:36:02] ENGINE Bus EXITED
> [19/Sep/2018:03:36:02] ENGINE Waiting for child threads to terminate...
> Push server created on address /run/user/0/woknotifications
> 
> *** Kimchi: Running feature tests ***
> Service Libvirtd running ...: True
> operation failed: no storage pools were found on host '127.0.0.1'
> NFS Target Probe support ...: True
> Fibre Channel Host support .: True
> Kernel VFIO support ........: True
> Network Manager running ....: True
> Memory Hotplug support .....: False
> *** Kimchi: Feature tests completed ***
> 
> Unknown device type: drm
> Unknown device type: drm
> Unknown device type: drm
> Failed to import plugin plugins.kimchi.Kimchi, error: internal error: process exited while connecting to monitor: Could not access KVM kernel module: Permission denied
> 2018-09-19T07:40:03.393803Z qemu-system-x86_64: failed to initialize KVM: Permission denied
> *** Ginger Base: Running capabilities tests ***
> System Report Tool ...: True
> Logging AptUpdate features.
> System Update Tool ...: True
> Repo Management Tool .: deb
> *** Ginger Base: Capabilities tests completed ***
> [19/Sep/2018:03:40:13] ENGINE Listening for SIGHUP.
> [19/Sep/2018:03:40:13] ENGINE Listening for SIGTERM.
> [19/Sep/2018:03:40:13] ENGINE Listening for SIGUSR1.
> [19/Sep/2018:03:40:13] ENGINE Bus STARTING
> [19/Sep/2018:03:40:14] ENGINE Started monitor thread 'Session cleanup'.
> [19/Sep/2018:03:40:14] ENGINE Serving on http://127.0.0.1:8010
> 
> *** Kimchi: Running dependable feature tests ***
> QEMU stream support .......: False
> Libvirt Stream Protocols ..: ['http', 'https', 'ftp', 'ftps', 'tftp']
> *** Kimchi: Dependable feature tests completed ***
> 
> [19/Sep/2018:03:40:16] ENGINE Bus STARTED
> [19/Sep/2018:03:44:42] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:44:42] ENGINE Bus STOPPING
> [19/Sep/2018:03:44:42] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:44:42] ENGINE Bus STOPPING
> [19/Sep/2018:03:44:42] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) already shut down
> [19/Sep/2018:03:44:42] ENGINE Stopped thread 'Session cleanup'.
> [19/Sep/2018:03:44:42] ENGINE Bus STOPPED
> [19/Sep/2018:03:44:42] ENGINE Bus EXITING
> [19/Sep/2018:03:44:42] ENGINE Bus EXITED
> [19/Sep/2018:03:44:42] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) shut down
> [19/Sep/2018:03:44:42] ENGINE No thread running for Session cleanup.
> [19/Sep/2018:03:44:42] ENGINE Bus STOPPED
> [19/Sep/2018:03:44:42] ENGINE Bus EXITING
> [19/Sep/2018:03:44:42] ENGINE Bus EXITED
> [19/Sep/2018:03:44:42] ENGINE Waiting for child threads to terminate...
> Push server created on address /run/user/0/woknotifications
> 
> *** Kimchi: Running feature tests ***
> Service Libvirtd running ...: True
> operation failed: no storage pools were found on host '127.0.0.1'
> NFS Target Probe support ...: True
> Fibre Channel Host support .: True
> Kernel VFIO support ........: True
> Network Manager running ....: True
> Memory Hotplug support .....: False
> *** Kimchi: Feature tests completed ***
> 
> Unknown device type: drm
> Unknown device type: drm
> Unknown device type: drm
> Failed to import plugin plugins.kimchi.Kimchi, error: internal error: process exited while connecting to monitor: Could not access KVM kernel module: Permission denied
> 2018-09-19T07:44:46.099538Z qemu-system-x86_64: failed to initialize KVM: Permission denied
> *** Ginger Base: Running capabilities tests ***
> System Report Tool ...: True
> Logging AptUpdate features.
> System Update Tool ...: True
> Repo Management Tool .: deb
> *** Ginger Base: Capabilities tests completed ***
> [19/Sep/2018:03:44:46] ENGINE Listening for SIGHUP.
> [19/Sep/2018:03:44:46] ENGINE Listening for SIGTERM.
> [19/Sep/2018:03:44:46] ENGINE Listening for SIGUSR1.
> [19/Sep/2018:03:44:46] ENGINE Bus STARTING
> [19/Sep/2018:03:44:46] ENGINE Serving on http://127.0.0.1:8010
> 
> *** Kimchi: Running dependable feature tests ***
> [19/Sep/2018:03:44:46] ENGINE Started monitor thread 'Session cleanup'.
> QEMU stream support .......: False
> Libvirt Stream Protocols ..: ['http', 'https', 'ftp', 'ftps', 'tftp']
> *** Kimchi: Dependable feature tests completed ***
> 
> [19/Sep/2018:03:44:47] ENGINE Bus STARTED
> [19/Sep/2018:03:46:03] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:46:03] ENGINE Bus STOPPING
> [19/Sep/2018:03:46:03] ENGINE Caught signal SIGTERM.
> [19/Sep/2018:03:46:03] ENGINE Bus STOPPING
> [19/Sep/2018:03:46:03] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) already shut down
> [19/Sep/2018:03:46:03] ENGINE Stopped thread 'Session cleanup'.
> [19/Sep/2018:03:46:03] ENGINE Bus STOPPED
> [19/Sep/2018:03:46:03] ENGINE Bus EXITING
> [19/Sep/2018:03:46:03] ENGINE Bus EXITED
> [19/Sep/2018:03:46:03] ENGINE HTTP Server cherrypy._cpwsgi_server.CPWSGIServer(('127.0.0.1', 8010)) shut down
> [19/Sep/2018:03:46:03] ENGINE No thread running for Session cleanup.
> [19/Sep/2018:03:46:03] ENGINE Bus STOPPED
> [19/Sep/2018:03:46:03] ENGINE Bus EXITING
> [19/Sep/2018:03:46:03] ENGINE Bus EXITED
> [19/Sep/2018:03:46:03] ENGINE Waiting for child threads to terminate...
> Push server created on address /run/user/0/woknotifications
> 
> *** Kimchi: Running feature tests ***
> Service Libvirtd running ...: True
> operation failed: no storage pools were found on host '127.0.0.1'
> NFS Target Probe support ...: True
> Fibre Channel Host support .: True
> Kernel VFIO support ........: True
> Network Manager running ....: True
> Memory Hotplug support .....: False
> *** Kimchi: Feature tests completed ***
> 
> Unknown device type: drm
> Unknown device type: drm
> Unknown device type: drm
> Failed to import plugin plugins.kimchi.Kimchi, error: internal error: process exited while connecting to monitor: Could not access KVM kernel module: Permission denied
> 2018-09-19T07:46:06.444978Z qemu-system-x86_64: failed to initialize KVM: Permission denied
> *** Ginger Base: Running capabilities tests ***
> System Report Tool ...: True
> Logging AptUpdate features.
> System Update Tool ...: True
> Repo Management Tool .: deb
> *** Ginger Base: Capabilities tests completed ***
> [19/Sep/2018:03:46:07] ENGINE Listening for SIGHUP.
> [19/Sep/2018:03:46:07] ENGINE Listening for SIGTERM.
> [19/Sep/2018:03:46:07] ENGINE Listening for SIGUSR1.
> [19/Sep/2018:03:46:07] ENGINE Bus STARTING
> [19/Sep/2018:03:46:07] ENGINE Serving on http://127.0.0.1:8010
> 
> *** Kimchi: Running dependable feature tests ***
> [19/Sep/2018:03:46:07] ENGINE Started monitor thread 'Session cleanup'.
> QEMU stream support .......: False
> Libvirt Stream Protocols ..: ['http', 'https', 'ftp', 'ftps', 'tftp']
> *** Kimchi: Dependable feature tests completed ***
> 
> [19/Sep/2018:03:46:07] ENGINE Bus STARTED
>

**Comment by saulhuerta** [comment]
Ok, I read log you shared.

I can see: 
- Details: cannot open directory '/var/lib/libvirt/images': No such file or directory
- Failed to import plugin plugins.kimchi.Kimchi, error: Storage pool default does not exist or is not active. 
- Failed to import plugin plugins.kimchi.Kimchi, error: internal error: process exited while connecting to monitor: Could not access KVM kernel module: Permission denied

I recommend you:
- Check you have installed the complete wok & kimchi dependencies.
- Reinstall kimchi plugin 
- Restart wokd service

Try  and share your error logs again please.

Regards

**Comment by OMEmu** [comment]
Kimchi's template.conf is likely incorrect. I had to edit mine to point to the correct storage pool (and I had run a standard installation).
`sudo virsh pool-list`
will give you all of the storage pools. I suspect that _default_ isn't one of them. _image_ is the default on my system, so I changed
`# Storage pool used to handle the guest disk`
`#pool = default`
to
`# Storage pool used to handle the guest disk`
`pool = image`

I worked this out from a comment by @alinefm in #1097
